File: MasEjemplos/OMP-solution/pitfalls/race2.f90
program calc1
      JLAST = -777
!$OMP PARALLEL DO LASTPRIVATE(JLAST)
      DO 10 J=1,1000
           JLAST = J
10    CONTINUE 
!$OMP END PARALLEL DO
      print *, JLAST
end program calc1

File: MasEjemplos/OMP-solution/pitfalls/race2.c
#include <stdio.h>

int main(void){
  int jlast=-777;
  int i;
#pragma omp parallel for lastprivate(jlast)
  for(i=1; i<1000; i++){
    jlast=i;
  }

  printf("%d\n",jlast);


  return 0;
}
